src/utils.c

```diff
@@ -0,0 +1,35 @@
+
+#include <stdarg.h>
+#define INCL_DOSSEMAPHORES

```

Comment:
No need in extra line and this header.
```suggestion
#define INCL_DOSSEMAPHORES
```